sv_pipeline/exome/load_data.py

```diff
@@ -106,18 +118,31 @@ def _parse_genes(genes):
         'field_name': GENES_FIELD,
         'format': _parse_genes,
     },
-    PREV_IDENTICAL_COL: {'field_name': 'prev_call', 'format': bool},
-    PREV_OVERLAP_COL: {'field_name': 'prev_overlap', 'format': bool},
+    PREV_IDENTICAL_COL: {'field_name': PREV_CALL_FIELD, 'format': bool},
+    PREV_OVERLAP_COL: {'field_name': PREV_OVERLAP_FIELD, 'format': bool},
     PREV_MISSING_COL: {'field_name': NEW_CALL_FIELD, 'format': lambda call: call not in {'NA', 'FALSE'}},
 }
+
 COL_CONFIGS.update({col: {'format': _parse_genes} for col in GENE_CONSEQUENCE_COLS.keys()})
 
+MERGED_CALL_COL_CONFIGS = copy.deepcopy(COL_CONFIGS)
+MERGED_CALL_COL_CONFIGS.update({
+    IS_LATEST: {'field_name': PREV_CALL_FIELD, 'format': lambda val: not BOOL_MAP[val.strip()]},
+    PREV_OVERLAP_COL: {'field_name': PREV_OVERLAP_FIELD, 'format': lambda val: False},
+    PREV_MISSING_COL: {'field_name': NEW_CALL_FIELD, 'format': lambda val: False},
+})
+
 CORE_COLUMNS = [CHR_COL, SC_COL, SF_COL, CALL_COL, IN_SILICO_COL]
 SAMPLE_COLUMNS = [
     START_COL, END_COL, QS_COL, CN_COL, NUM_EXON_COL, GENES_COL, DEFRAGGED_COL, PREV_IDENTICAL_COL, PREV_OVERLAP_COL,
     PREV_MISSING_COL,
 ] + list(GENE_CONSEQUENCE_COLS.keys())
+
+MERGED_CALL_SAMPLE_COLUMNS = SAMPLE_COLUMNS + [IS_LATEST]
```

Comment:
the default configuration should be the one thats in the main config, and since merged is now the default you should update the main `COL_CONFIGS`/ `SAMPLE_COLUMNS ` and maintain the joint called configs as separate

Reply:
Okay